src/charm.py

```diff
@@ -37,7 +37,7 @@
 from utilities import bytes_with_unit_to_kib, execute_command, get_env_var, retry
 
 if TYPE_CHECKING:
-    from ops.model import JsonObject  # pragma: no cover
+    from ops.model import JsonObject  # type: ignore
```

Comment:
The latest ops library seems to have replaced JsonObject to typing.Any, I've implemented the changes in [yq PR](https://github.com/canonical/github-runner-operator/pull/77/files), :D just a heads up in case the linting fails

Reply:
Ok. It should be fix once that PR is merged. Thanks.